src/odbc.cpp

```diff
@@ -325,7 +325,7 @@ class ConnectAsyncWorker : public ODBCAsyncWorker {
         return_code = SQLSetConnectAttr(
           hDBC,                                   // ConnectionHandle
           SQL_ATTR_CONNECTION_TIMEOUT,            // Attribute
-          (SQLPOINTER) size_t(options->connectionTimeout), // ValuePtr
+          (SQLPOINTER) intptr_t(options->connectionTimeout), // ValuePtr
```

Comment:
Seems like an odd mix of casts. I guess technically the intptr_t is not a cast, but the constructor. Might be worth a macro INT2PTR or something.

Reply:
Oops, I actually meant to do two casts. If I just cast to an SQLPOINTER, it complains that the 

> warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]

So if I cast to the correctly-sized int first, the fixes the warning. But if I just cast to `intptr_t` and not `SQLPOINTER` as well, I get an actual compiler error:

> error: invalid conversion from ‘intptr_t {aka long int}’ to ‘SQLPOINTER {aka void*}